docs/release/3.5/release-3-5-33.md

```diff
@@ -0,0 +1,132 @@
+title: OSG 3.5.33
+
+OSG Software Release 3.5.33
+===========================
+
+**Release Date:** 2021-04-01  
+**Supported OS Versions:** EL7, EL*
+
+!!!tip "Want faster access to production-ready software?"
+    OSG 3.5 offers a rolling release repository where packages are added as soon as they pass acceptance testing.
+    To install packages from this repository, enable `[osg-rolling]` in `/etc/yum.repos.d/osg-rolling.repo`:
+
+        [osg-rolling]
+        ...
+        enabled=1
+
+    Or for one-time installations, append the following to your `yum` command:
+
+        --enablerepo=osg-rolling
+
+Summary of Changes
+------------------
+
+!!! bug "Known issues with XRootD 5.1.1"
+    -   The XRootD team is investigating memory leaks in the HTTP Third-Party Copy (HTTP-TPC) use case related to
+        `libcurl` and `NSS`.
+        **As a result, XRootD 5.1.1 is not recommended for sites supporting HTTP-TPC.**
+    -   [Incompatibility with the multi-user plugin](https://github.com/opensciencegrid/xrootd-multiuser/issues/21):
+        users of the XRootD multi-user plugin will be unable to update to XRootD 5.1.x until a fixed version
+        of XRootD multi-user is released into the OSG repositories
+    -   In some cases, XCaches using the Rucio plug-in may crash due to malformed URLs generated by the plug-in.
+
+This release contains the following changes to the `osg-upcoming` repository:
+
+-   `XRootD 5.1.1` (EL7 only): an update from the previously available version, XRootD 5.0.2
+    -  See [this section](../updating-to-osg-35.md#updating-to-xrootd-5) for detailed update instructions
+    -  The XRootD SciTokens plug-in has been merged into the XRootD code-base so its version now matches the `xrootd`
```

Comment:
```suggestion
    -  The XRootD SciTokens plug-in (`xrootd-scitokens`) has been merged into the XRootD code-base so its version now matches the `xrootd`
```
If you're going to use the RPM name for XRootD, you should use it for the plugin too.